inputs/LoginForm.tsx

```diff
@@ -0,0 +1,52 @@
+import { Text, View, TextInput, TouchableOpacity } from 'react-native';
+import React, { useState } from 'react';
+import styles from '../styles';
+
+export default function LoginForm({}) {
+    const [email, setEmail] = useState('');
```

Comment:
This is adding extra logic separately when we could combine everything in one object state, as we did in the inputs lesson, to maintain readability. How to set it as one state and then update was in the inputs lesson
```tsx
const [form, setForm] = useState({
    email: '',
    password: ''
  })
  ```